src/components/ifxToggle/ifxToggle.tsx

```diff
@@ -0,0 +1,28 @@
+// pill-toggle.tsx
+import { Component, Prop, h, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-toggle',
+  styleUrl: 'ifxToggle.scss',
+  shadow: true,
+})
+export class ifxToggle {
+  @Prop({ mutable: true }) checked: boolean = false;
```

Comment:
`Outline` variant is missing.

Reply:
There is no Outline Variant in Figma? (maybe i'm blind)